App.js

```diff
@@ -1,11 +1,37 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, FlatList, Image, ScrollView, AppRegistry, Dimensions} from 'react-native';
```

Comment:
Do change the `App.js` file. Instead, you should make sure that it looks the same as the `master` branch's version. If you want to see how your `Register` component works, import it, and use it as a React Native component. i.e. `<Register />`